lmfdb/number_fields/number_field.py

```diff
@@ -457,24 +457,28 @@ def render_field_webpage(args):
         loc_alg = ''
         for j in range(npr):
             if ramified_algebras_data[j] is None:
-                loc_alg += '<tr><td>%s<td colspan="7">Data not computed'%str(ram_primes[j]).rstrip('L')
+                loc_alg += '<tr><td>$%s$<td colspan="7">Data not computed'%str(ram_primes[j]).rstrip('L')
```

Comment:
Add `</td>` and `</tr>` tags here; I'm also not sure why the `rstrip('L')` is here.

Reply:
The rstrip is there to remove an L if it appears when a large integer is converted to a string by python.